### ISSUE TITLE
Warning: useLayoutEffect does nothing on the server

### ISSUE THREAD

**Issue by beckend** [xaviergonz/mobx-react-component#5]
```javascript
@mobxComponent()
export class MyComponent extends MobxComponent<IMyComponentProps> {...}
```

SSR warning
`Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://fb.me/react-uselayouteffect-ssr for common fixes.
`

**Comment by xaviergonz** [comment]
Should be fixed in 0.51.0, if not please reopen the issue. Thanks!

**Comment by xaviergonz** [comment]
Ah, when using SSR remember to use `useMobxStaticRendering(true);` (just added that to the readme)
